mmtrack/models/trackers/byte_tracker.py

```diff
@@ -0,0 +1,313 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import lap
+import numpy as np
+import torch
+from mmcv.runner import force_fp32
+from mmdet.core import bbox_overlaps
+
+from mmtrack.models import TRACKERS
+from .base_tracker import BaseTracker
+
+
+@TRACKERS.register_module()
+class ByteTracker(BaseTracker):
+    """Tracker for ByteTrack.
+
+    Args:
+        high_det_score (float): Threshold of the first matching detection bbox.
+        low_det_score (float): Threshold of the second matching detection bbox.
+        init_track_score (float): Threshold of initializing a new tracklet.
+        weight_iou (bool): Whether using detection scores to weight IOU.
+        first_match_iou_thr (float): Threshold of the first matching threshold.
```

Comment:
`match_iou_thrs = [A, B]`?, or `match_iou_thrs = dict(top=1, medium=2, tentative=3 )`

Reply:
Let's take `match_iou_thrs = dict(high=1, low=2, tentative=3 )`

Reply:
And, `obj_score_thrs = dict(hight=1, low=1)`